packages/host/tests/helpers/adapter.ts

```diff
@@ -0,0 +1,299 @@
+import {
+  Loader,
+  LocalPath,
+  RealmAdapter,
+  RealmPaths,
+  baseRealm,
+  createResponse,
+} from '@cardstack/runtime-common';
+
+import {
+  FileRef,
+  Kind,
+  Realm,
+  TokenClaims,
+  UpdateEventData,
+} from '@cardstack/runtime-common/realm';
+
+import { WebMessageStream, messageCloseHandler } from './stream';
+
+import { createJWT, testRealmURL } from '.';
+
+interface Dir {
+  kind: 'directory';
+  contents: { [name: string]: File | Dir };
+}
+
+interface File {
+  kind: 'file';
+  content: string | object;
+}
+
+type CardAPI = typeof import('https://cardstack.com/base/card-api');
+
+class TokenExpiredError extends Error {}
+class JsonWebTokenError extends Error {}
+
+interface TestAdapterContents {
+  [path: string]: string | object;
+}
+
+let shimmedModuleIndicator = '// this file is shimmed';
+
+export class TestRealmAdapter implements RealmAdapter {
```

Comment:
I extracted `TestRealmAdapter` from helpers because the class is not small and it's easier to handle in its own file